nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/api/scenarios/NBCLIScenarioPreprocessor.java

```diff
@@ -437,28 +433,43 @@ public static List<String> getScripts(boolean defaultIncludes, String... include
 
     }
 
-    public static Map<String, String> matchTemplates(String line, Map<String, String> templates) {
-        Matcher matcher = templatePattern.matcher(line);
-
-        while (matcher.find()) {
-            String match = matcher.group(1);
-
-            Matcher innerMatcher = innerTemplatePattern.matcher(match);
-            if (innerMatcher.find()) {
-                String innerMatch = innerMatcher.group(1);
-                templates = matchTemplates("TEMPLATE(" + innerMatch + ")", templates);
-                String resolvedInner = templates.getOrDefault(innerMatch.split("[,:]")[0], "");
-                match = match.replace("TEMPLATE(" + innerMatch + ")", resolvedInner);
-            }
+    //    private static final Pattern templatePattern = Pattern.compile("TEMPLATE\\((.+?)\\)");
+    //    private static final Pattern innerTemplatePattern = Pattern.compile("TEMPLATE\\((.+?)\\)");
+    private static final Pattern templatePattern2 = Pattern.compile("<<(.+?)>>");
```

Comment:
The `<<name:default>>` pattern should be deprecated with a warning. We introduced the `TEMPLATE(name,default)` format because it didn't throw off syntax checkers, etc.

Reply:
Deprecation warning on the use of <<name:default>> is a new requirement.

Reply:
This is now checked.